src/libraries/System.Net.Http/src/System/Net/Http/HttpHandlerDefaults.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading;
+
+namespace System.Net.Http
+{
+    /// <summary>
+    /// Additional default values used used only in this assembly.
+    /// </summary>
+    internal static partial class HttpHandlerDefaults
+    {
+        public static readonly TimeSpan DefaultKeepAlivePingTimeout = TimeSpan.FromSeconds(20);
```

Comment:
JIT will optimize this better if you do:

```suggestion
        public static TimeSpan DefaultKeepAlivePingTimeout => TimeSpan.FromTicks(TimeSpan.TicksPerSecond * 20);
```

Reply:
Why the property? To make it lazy?

Reply:
the readonly field will take up memory and require initialization.
the property will be optimized 100% into a constant `mov rax, 1234h; ret`.

Reply:
@scalablecory It comes from other [default timeouts](https://github.com/dotnet/runtime/pull/40257/files/20f44cf98a4de5734deb1e1d146cbb891c2a9cd0#diff-1ef6a69559626aa1f87616cd3f0106c1R17) defined on the class (but in a different file due to being linked in WinHttpHandler).

If it gets changed here, it might cause confusion why is here used different approach.

Reply:
I'll add the change to the rest values